client/src/componets/Details/Details.module.css

```diff
@@ -1,66 +1,141 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=MuseoModerno:wght@400;700&display=swap');
 
+.n .loading {
+  display: block;
+  margin: 0px auto;
+}
 
-.name{
+.container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 10px;
+}
+
+.name {
   font-family: 'Montserrat', sans-serif;
-  text-align:center;
+  margin-left: 10%;
+  margin-bottom: 20px;
 }
-.img{
+
+.img {
   height: 300px;
-  border-radius: 20px;
+  border-radius: 10px;
   width: 400px;
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  
+  margin-left: 5%;
+  margin: 10px 0 15px 60px;
+
 }
-.description__p{
- font-family: 'Montserrat', sans-serif;
- padding-left: 40px;
- padding-right: 40px;
- padding-top: 5px;
- text-align: center;
+
+.container__info {
+  display: grid;
+
 }
-.description__text{
+
+.fecha .info {
+  margin-left: 5%;
+  margin: 10px 20px 15px 60px;
+}
+
+.description__p {
   font-family: 'Montserrat', sans-serif;
-  padding-left: 40px;
-  padding-right: 40px;
-  padding-top: 10px;
 }
 
-.rating{
+.description__text {
   font-family: 'Montserrat', sans-serif;
-  padding-top: 20px;
-  text-align:center;
+  margin-left: 100px;
+  text-align: justify;
+  margin-bottom: 40px;
 }
-.lanzamiento{
+
+.rating {
   font-family: 'Montserrat', sans-serif;
-  padding-top: 20px;
-  text-align:center;
+  color: #FE7F3C;
 }
+
+.lanzamiento {
```

Comment:
You are not using english. Maybe you can create an issue for this one instead of changing it for this PR.

Reply:
@andresyrg16  I'm still waiting for the issue creation